src/core/utils/LockHashUtils.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { sha3_256 } from 'js-sha3';
+import { Convert } from '../format/Convert';
+
+/**
+ * Hash utilities for SecretLock hashing
+ */
+export class LockHashUtils {
```

Comment:
We should add a few static test cases like

https://github.com/nemtech/symbol-sdk-java/blob/33e17792ce2926f601ba2b50f1982d7e5d8be030/sdk-core/src/test/java/io/nem/symbol/core/crypto/HashesTypescriptValidationTest.java#L40

Se we are sure we are hashing the same way

Reply:
Not sure we need to do it here as the OP_ hashes are specific for SecretLock. Also we don't have a test vector for these. So I think minimal unit test + integration test should be enough?

Reply:
We can create our own test vectors in the SDKs, if there are too many hash types, just focus on the ones used.